python/dgl/data/dgl_dataset.py

```diff
@@ -221,6 +220,15 @@ def _get_hash(self):
         hash_func.update(str(self._hash_key).encode("utf-8"))
         return hash_func.hexdigest()[:8]
 
+    def _get_hash_url(self):
+        """Compute the hash of the url"""
+        if self._url is None:
+            return ""
+        else:
+            hash_func = hashlib.sha1()
+            hash_func.update(str(self._url).encode("utf-8"))
+            return "_" + hash_func.hexdigest()[:8]
```

Comment:
the `"_" +` should not be appended here since the function is named. _get_hash_url

Reply:
renamed to `_get_hash_url_suffix`